src/frontend/js/components/LtiConsumer/index.tsx

```diff
@@ -45,7 +42,8 @@ const LtiConsumer = ({
         name="lti_iframe"
         title={url}
         src={url}
-        allow="microphone *; camera *; midi *; geolocation *; encrypted-media *"
+        allow="microphone *; camera *; midi *; geolocation *; encrypted-media *; fullscreen *"
```

Comment:
Should we remove geolocation and midi?

Reply:
I hesitated to delete them, but this component should be support all kind of LTI tool and may be one can require geolocation and midi?

Reply:
> I hesitated to delete them, but this component should support all kind of LTI tool and may be one of them can require geolocation and midi?

That's why I've keeped them, but I don't remember where I found them.